packages/expo-linear-gradient/src/NativeLinearGradient.web.tsx

```diff
@@ -16,6 +16,13 @@ type State = {
 
 type Point = [number, number];
 
+function hexStringFromProcessedColor(color: number) {
```

Comment:
Move this function below the class so that the more salient code comes first in the file.